<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsDialog</class>
 <widget class="QDialog" name="ToolsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QSlider" name="radiusSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="radiusLabel">
     <property name="text">
      <string>Tool radius :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="sweepButton">
       <property name="text">
        <string>Sweep</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defInfButton">
       <property name="text">
        <string>Inflate/Deflate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="twistButton">
       <property name="text">
        <string>Twist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="radiusSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
